docs/core/diagnostics/cli-tools/dotnet-counters.md

```diff
@@ -0,0 +1,137 @@
+---
+title: dotnet-counters - .NET Core
+description: Installing and using the dotnet-counter command-line tool.
+author: sdmaclea
+ms.author: stmaclea
+ms.date: 08/21/2019
+---
+# dotnet-counters
+
+## Intro
+
+`dotnet-counters` is a performance monitoring tool for ad-hoc health monitoring or first-level performance investigation. It can observe performance counter values that are published via the `EventCounter` [API](https://docs.microsoft.com/dotnet/api/system.diagnostics.tracing.eventcounter). For example, you can quickly monitor things like the CPU usage or the rate of exceptions being thrown in your .NET Core application to see if there's anything suspicious before diving into more serious performance investigation using `PerfView` or `dotnet-trace`.
+
+## Install dotnet-counters
+
+To install the latest release version of the `dotnet-counters` [NuGet package](https://www.nuget.org/packages/dotnet-counters):
+
+```
+dotnet tool install --global dotnet-counters
+```
+
+For details and other options, see [Installing the diagnostics tools](installing.md).
+
+> [!NOTE]
+> At the time of this writing, the `dotnet-counters` tool has not been released. The prerelease version can be installed using:
+>
+> ```
+> dotnet tool install --global dotnet-counters --version 3.0.0-preview8.19412.1
+> ```
+>
+
+## Using dotnet-counters
+
+*SYNOPSIS*
+
+```
+dotnet-counters [--version]
+                [-h, --help]
+                <command> [<args>]
+```
+
+*OPTIONS*
+
+```
```

Comment:
If this is supposed to be a reference for this tool, I think it should probably follow the pattern we already use in the .NET Core CLI docs. e.g. https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-build, for consistency, but with the additional information about how to install it.

Also, nothing will get localized since all the text is inside a code block.

This applies to the other topics as well.